src/main/java/stringcalculator/CalculatorNumbers.java

```diff
@@ -0,0 +1,27 @@
+package stringcalculator;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CalculatorNumbers {
+    public static final String DEFAULT_DELIMETER = ":|,";
+    private List<Integer> calculatorNumbers;
+
+    public CalculatorNumbers(String operand) {
+        Matcher matcher = Pattern.compile("//(.)\n(.*)").matcher(operand);
+        String delimeter = DEFAULT_DELIMETER;
+        if (matcher.find()) {
+            delimeter = matcher.group(1);
+            operand = matcher.group(2);
+        }
+        calculatorNumbers = Arrays.stream(operand.split(delimeter)).map(Integer::parseInt).toList();
+    }
+
+    public int sum() {
+        return calculatorNumbers.stream().mapToInt(Integer::intValue).sum();
```

Comment:
체이닝 메서드를 한줄로 표현하기보다는 한줄에 하나의 메서드만 호출되도록한다면 조금 더 코드의 가독성을 높일 수 있을 것 같아요. 

```suggestion
        return calculatorNumbers.stream()
            .mapToInt(Integer::intValue)
            .sum();
```

Reply:
넵넵 변경완료했습니다!